packages/keystone/bin/cli.js

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env node
+
 const arg = require('arg');
 const path = require('path');
 const globby = require('globby');
+try {
+  require('@keystone-alpha/build-field-types').hook(path.join(__dirname, '..', '..', '..'));
```

Comment:
What is this doing?

Reply:
It sets up require hooks so when you import from an entrypoint built with `build-field-types` it will import the source file and run Babel on it and it's dependencies. This means that you get live reload for the client-side stuff and for the server-side stuff, you don't have to run a build or watch process.

I'm not 100% sure if it's a good idea because it feels a bit hacky but it seems to work pretty well so maybe it's okay ¯\\\_(ツ)_/¯.

I had another idea to sort of solve this problem. adding a command called `dev` to build-field-types/preconstruct which would create symlinks for all the entrypoints from where the dist files would be to the appropriate source file. I think it might be nice idea since then you don't need tool-specific aliases and everything just works.

Babel still needs to be run on the source files.

(this is just my process of thinking through this)

Maybe, instead of symlinks, actual files could be written with something like this

```jsx
// babel require hook (not @babel/register, a custom one because we need to be able to specifically unregister the one we register which @babel/register doesn't allow)
module.exports = require('../source-file')
// unregister babel require
```

Though then it still uses a require hook and that would be a worse experience for Jest and it wouldn't work in webpack.(though i suppose the module dev file could just re-export or be a symlink and then it would work for webpack)

So maybe the best thing to do would be symlinks + a require hook for the node use case so the tool-specific aliases are unnecessary.